.github/workflows/nodejs-ci.yml

```diff
@@ -16,11 +16,13 @@ jobs:
         node-version: [16.x]
     steps:
       - uses: actions/checkout@v2
+      - uses: browser-actions/setup-chrome@latest
```

Comment:
This step setups the Chrome browser that is used by the Karma test runner in the `npm run test:ci` step.